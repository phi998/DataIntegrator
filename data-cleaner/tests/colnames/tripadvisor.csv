Name,Unknown,Location,Unknown,Location,Rating,other,Location,Address,Rating,Rating,funzionalità,Phone number,Rating,Rating,Rating,Rating,Rating,Rating,Rating,Rating,Rating,Name,Rating,Rating,Name,Rating,Rating,Date,Name,UnknownName,Unknown,Location,Unknown,Location,Rating,Rating,Rating,Address,Number of Views,Rating,funzionalità,Phone number,Rating,Rating,Rating,Rating,Rating,Rating,Rating,Rating,Rating,Name,Rating,Rating,Name,Rating,Name,Date,Name,Unknown
