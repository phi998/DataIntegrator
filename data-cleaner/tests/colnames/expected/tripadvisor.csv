Name,.,Location,.,Location,.,Rating,Location,.,Rating,Rating,.,Phone number,.,.,.,.,.,.,.,.,.,.,.,.,.,Rating,.,Date,.,.
